website/docs/release_notes.md

```diff
@@ -754,7 +940,7 @@ It is now possible to run Scala CLI in offline mode for the cases when you don't
 to make any network requests for whatever reason.
 This changes Coursier's cache policy to `LocalOnly`, preventing it from downloading anything.
 
-```bash 
+```bash ignore
```

Comment:
ignoring this, since the `.` catches sources it shouldn't in documentation tests